plugins/modules/sorcery.py

```diff
@@ -641,10 +731,13 @@ def main():
     if params['update']:
         update_sorcery(module)
 
-    if params['update_cache'] or params['state'] == 'latest':
+    if params['name'] and params['repository']:
+        manage_grimoires(module)
+
+    if params['update_cache']:
         update_codex(module)
```

Comment:
Now this is no longer called if `params['state'] == 'latest'`. Is that intentional?

Reply:
@felixfontein yeah, this was an oversight in the original version of the module that mixes in 2 actions, while in fact there should be flexibility to control them separately to avoid unexpected results. To me that still falls under minor changes and I added a note to the fragments.